examples/simple-platform/user-walk-through.md

```diff
@@ -0,0 +1,107 @@
+# User Walk-through of the Platform Example
+
+## Create an account and complete the OAuth flow with GitHub
+
+1. Open the accounts app in your browser at `http://localhost:3025`.
+
+1. Open the browser's developer tools to the console tab and verify that you
+see the `interbit-middleware/CHAIN_BLOCKING` status logged to the console for
+the `accountsPrivate` chain.
+
+   The Interbit middleware prints a number of messages to the console so you
+can see, for instance, when chains have loaded and actions have been
+dispatched. The account app is fully up and running when you see the above
+message. At this point, you can dispatch actions to your
+private account chain.
+
+1. Navigate to the `create-account` page by clicking on the **Create Account
+/ Sign in** link in the header.
+
+1. Create an authenticated user account by clicking the **Create Account**
+button, checking that you have read the disclaimer, and clicking the
+**Continue** button in the modal dialog.
+
+1. Sign in to your GitHub account. If you're already logged into your GitHub
+account, your personal account page is displayed. Otherwise, the GitHub
+sign-in page appears. Enter your GitHub credentials and sign in to your
+account. You should then be redirected back to the accounts app's personal
+account page.
+
+   You should see a number of chain actions being dispatched in the browser
+console on the accounts app tab. Specifically, after the action type
+`app-account-my-account/COMPLETE_AUTHENTICATION` is dispatched, the
+`chainState.profile` property is updated with your GitHub identity
+information. You can also verify that the OAuth flow was successful if you see
+your GitHub username appear in the header.
+
+   You have now created an Interbit account. You can make changes to your
+personal account information and see how the new blocks are added to your
+private account chain with each change.
```

Comment:
Is there one chain per Interbit account? If so, we should say `...to the private blockchain containing your account information`. If not, we should replace `account` with `accountsPrivate`.